tdrs-backend/tdpservice/search_indexes/models.py

```diff
@@ -1,9 +1,12 @@
 """Models representing parsed data file records submitted to TDP."""
 
 from django.db import models
-
-# Create your models here.
-
+from tdpservice.data_files.models import DataFile
+# from .parsers.validators import (
+#    record_validator,
+#    rpt_month_year as month_year_validator,
+# )
+# from django.core.validators import RegexValidator
```

Comment:
Will be taken care of in #1354 ; might remove mentions of validators for cleanliness.